tests/e2e/configurer/chain/commands.go

```diff
@@ -770,6 +777,7 @@ func (n *NodeConfig) ParamChangeProposal(subspace, key string, value []byte, cha
 		Deposit: "625000000uosmo",
 	}
 	proposalJson, err := json.Marshal(proposal)
+	fmt.Println("marshalling output", proposalJson, err)
```

Comment:
```suggestion
```

Reply:
forgot to remove after testing. ty for removing this.